controllers/transactions.js

```diff
@@ -0,0 +1,36 @@
+const transactionsModel = require('../models/transaction')
+const { getUserId } = require('../utils/users')
+
+/**
+ * Collects all transactions and sends only required data for username spicified in url
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const fetch = async (req, res) => {
+  try {
+    const userId = await getUserId(req.params.username)
+    if (userId) {
+      const n = req.query.n || 10
+      const o = req.query.o || 'DESC'
+      const data = await transactionsModel.fetch(userId, n, o)
```

Comment:
Now I'm seeing the problem. If we keep using alphabets, it might become difficult to keep track of what's what.
`size` instead of `n` and `order` instead of `o` would make more sense, right? We wouldn't have to explicitly mention in the docs what's doing what
Apologies for the inconvenience.

Reply:
renamed according to purpose